### PR TITLE
[SPARK-5527] Improvements to standalone doc -explain how to sync cluster conf file

### PR DESCRIPTION
**Pull request by lazyman500**
Improvements to standalone doc -explain how to sync cluster conf file

### CODE REVIEW

File: docs/spark-standalone.md [hunk 1]
```diff
@@ -93,6 +93,10 @@ You can optionally configure the cluster further by setting environment variable
 <table class="table">
   <tr><th style="width:21%">Environment Variable</th><th>Meaning</th></tr>
   <tr>
+    <td><code>SPARK_MASTER</code></td>
```
Comment (srowen) [inline docs/spark-standalone.md]: Isn't this deprecated though?  I don't know that this is supposed to be documented.

Comment (lazyman500) [inline docs/spark-standalone.md]: Thanks for giving suggestion. Configuration file  synchronization  is a very common requirement, why not recommend? Do we need add some comment to explain it ?

Comment (srowen) [inline docs/spark-standalone.md]: This property is not specifically about configuration files, and is deprecated. You would set `--master` now. That's why I say documenting it doesn't sound like the right thing, right?

**Comment by andrewor14** [comment]
@lazyman500 this seems like a highly specific use case that was intended to be used internally. I bet even most committers for instance don't know about this option. For this reason I would recommend that we close this issue.

**Comment by lazyman500** [comment]
okey! thanks

**Pull request closed without merging**